frontend/src/app/main/ui/workspace.cljs

```diff
@@ -125,7 +125,10 @@
 (mf/defc workspace-loader
   []
   [:div {:class (stl/css :workspace-loader)}
```

Comment:
let's also check this wrapper

Reply:
I need to keep a wrappper on the full page loaders because I can't set width and height without setting those for the element dimensions.
Wrapper needs to be 100vh height and 100vw width but the loader has 64px height and 100 px width.